californium-core/src/main/java/org/eclipse/californium/core/network/CoapEndpoint.java

```diff
@@ -411,6 +411,7 @@ public void execute(Runnable command) {
 		LOGGER.info("{} uses {}", new Object[] { getClass().getSimpleName(), endpointContextMatcher.getName() });
 
 		if (CoAP.isTcpProtocol(connector.getProtocol())) {
+
```

Comment:
Please, try to omit changing files only for formatting.

Reply:
I will remove this.